sass/_t3_addons.sass

```diff
@@ -564,6 +564,12 @@ kbd
   padding: 0.2em 0.5em
   text-shadow: 0 1px 0 #fff
 
+// 2018-01-07
+// https://github.com/TYPO3-Documentation/t3SphinxThemeRtd/issues/96
+code.guilabel,
+guilabel
```

Comment:
This tag does not exist, `kbd` exists: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd while there is no https://developer.mozilla.org/en-US/docs/Web/HTML/Element/guilabel.

Therefore we might remove this selector and only add `code.guilabel`?

Reply:
kbd is different from guilabel. kbd describes keyboard strokes. We already have a style for that. guilabel describes text used in GUI, e.g. menu entries. I based my patch on https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html 

I had been meaning to add a style so it would be possible to do something like 

    :guilabel:`ADMIN TOOLS > Settings`.

If I understand it correctly, a style must be added for the textrole "guilabel" so you can use it inline, as described in  https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/InlineCode.html#how-to-semantically-markup-specific-text

Reply:
But this is `sass` which is compiled to CSS. And there is no html element `guilabel` which is styled here.

I'm not talking about whether to style or use `guilabel` or not, but that this sass will not do anything.